portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
```

Comment:
Please add doc for (public) methods

Reply:
Added! I would like to ask if the standard placement of the docstring is on top of the method name? or below it?
I saw that in the python doc-strings it's below the method name: https://www.geeksforgeeks.org/python-docstrings/#:~:text=Python%20Docstrings,a%20specific%20segment%20of%20code.

Reply:
Yes for js it's on top of the method.

Reply:
thank you for clarifying :)